### ISSUE TITLE
java.util.NoSuchElementException: No value present

### ISSUE THREAD

**Issue by Kishanthan** [ballerina-platform/ballerina-lang#29526]
**Description:**
Following error was observed when the cursor was the position as shown in the image.
<img width="803" alt="Screenshot 2021-03-24 at 00 01 27" src="https://user-images.githubusercontent.com/690559/112199367-1a28c680-8c34-11eb-930c-868a93d47fd6.png">

Error
```
[Error - 11:59:38 PM] Operation 'text/definition' failed! {uri: '/Users/kishanthan/WSO2/work/projects/ballerina-projects/test_hackathon/msmp_with_nats/publisher.bal', [11:18], error: 'No value present'}
java.util.NoSuchElementException: No value present
	at java.base/java.util.Optional.orElseThrow(Optional.java:382)
	at org.ballerinalang.langserver.util.definition.DefinitionUtil.getLocation(DefinitionUtil.java:82)
	at org.ballerinalang.langserver.util.definition.DefinitionUtil.getDefinition(DefinitionUtil.java:61)
	at org.ballerinalang.langserver.BallerinaTextDocumentService.lambda$definition$3(BallerinaTextDocumentService.java:257)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by nadeeshaan** [comment]
As per the error log, this is because the goto definition request has been triggered for standard library module content (`prinntln` method). This is yet to be implemented and being captured with #29558

**Comment by nadeeshaan** [comment]
Fixed with #29744. Hence closing the issue
